content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -153,29 +153,29 @@ Webhook conversion is introduced in kubernetes 1.13 as an alpha feature. To use
 {{< /note >}}
 
 The above example has a None conversion between versions which only sets the `apiVersion` field
-on conversion and does not change the rest of the object. API server also supports webhook conversions that
-calls an external service in case of a conversion is required, For example when:
+on conversion and does not change the rest of the object. The API server also supports webhook
+conversions that call an external service in case a conversion is required. For example when:
 
 * custom resource is requested in a different version than stored version.
 * Watch is created in one version but the changed object is stored in another version.
 * custom resource PUT request is in a different version than storage version.
 
-To cover all of these cases and to optimize conversion by the API server, the conversion requests contain as many objects as possible in order to minimize the calls to the webhook. These conversions should happened independently. The grouping of custom resources does not represent any relation between them.
+To cover all of these cases and to optimize conversion by the API server, the conversion requests may contain multiple objects in order to minimize the external calls. The webhook should perform these conversions independently.
 
 ### Write a conversion webhook server
 
 Please refer to the implementation of the [custom resource conversion webhook
-server](https://github.com/kubernetes/kubernetes/blob/v1.13.0-beta.1/test/images/crd-conversion-webhook/main.go)
+server](https://github.com/kubernetes/kubernetes/tree/v1.13.0/test/images/crd-conversion-webhook/main.go)
```

Comment:
LGTM, 👍, except for 404 on the link,
https://github.com/kubernetes/kubernetes/tree/v1.13.0/test/images/crd-conversion-webhook/main.go

@mbohlool, I was unable to get to this link from the preview page.
nit: There is a note on the page that should capitalize Kubernetes v1.13.
nit: Another note: When using ClientConfig.service ... where or what does this apply to, or is it unrelated to sample spec?

Reply:
the link will be valid when the 1.13.0 release is tagged

Reply:
👍

Reply:
capitalized text kubernetes -> Kubernetes.
About ClientConfig.service, it is related to the sample as the sample has `ClientConfig.service`. The use of https is forced as to another note and here we are just stating what name should the server certificate be valid for, for this config to work.